lib/core/src/lib/datatable/components/datatable/datatable.component.ts

```diff
@@ -333,6 +334,10 @@ export class DataTableComponent implements OnInit, AfterContentInit, OnChanges,
         if (this.isPropertyChanged(changes['sorting'])) {
             this.setTableSorting(changes['sorting'].currentValue);
         }
+
+        if (multiselectChanges.currentValue === false) {
```

Comment:
I think you could have a problem if `changes['multiselect']` is `undefined` right?